src/index.ts

```diff
@@ -13,6 +13,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import {GoogleAuth} from './auth/googleauth';
+
 export {Compute} from './auth/computeclient';
 export {GoogleAuth} from './auth/googleauth';
```

Comment:
nit: avoid second reference to './auth/googleauth'. Export in the newly added stanza below.